lib/defs.js

```diff
@@ -56,6 +62,17 @@ const
           .replace(/"/g, '\\"') // escape double quotations
         return `"${ val }"`
       }
+    },
+    'TEXT': {
+        type: 'TEXT',
+            convert: val => {
+            val = (val || '')
+                .toString()
+                .replace(/\\/g, '\\\\') // escape backslashs
+                .replace(/"/g, '\\"') // escape double quotations
+                .replace(/[^\x00-\x7F]/g, "") // remove non ascii characters
```

Comment:
@odedelharar thanks. I've checked the spec of MySQL: `\0` `\b` `\t` `\n` `\r` `\x1a` supposed to be allowed. Actually `sqlstring` module care about those control chars.
https://github.com/mysqljs/sqlstring/blob/master/lib/SqlString.js#L2

At this point, I'd like to remove control chars except above, and escape them by `sqlstring`.

The meaning of *Non-standard* depends on what charset we use, I think.
If you need another custom filter, feel free to send a new PR 😉